.changeset/cuddly-months-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/css": patch
```

Comment:
Isn't this technically a breaking change?

Reply:
Technically you could say so, but I'm operating under this assumption:

> We haven't begun to use this anywhere and it's not documented on the site yet, so I'm classifying this as a patch.

The code was added to this repository specifically for the view component which isn't complete yet.